apps/src/gamelab/GameLab.js

```diff
@@ -1482,6 +1512,31 @@ GameLab.prototype.captureInitialImage = function () {
   captureThumbnailFromCanvas(document.getElementById('defaultCanvas0'));
 };
 
+/**
+ * Log some performance numbers to firehose
+ */
+GameLab.prototype.reportMetrics = function () {
```

Comment:
Should we sample the data at all, or fine to log times for every run by every user?

Reply:
Yeah sounds good. Sampling 5% of page loads.